test/stringClassSpec.js

```diff
@@ -48,6 +46,10 @@ describe('isQuestion', () => {
   it('should return false if string is not a question', () => {
     expect('rhy$th#?m'.isQuestion()).toBeFalsy();
   });
+
+  it('should return false if string contains no word character', () => {
+    expect('???'.isQuestion()).toBeFalsy();
+  });
 });
 
 describe('Words', () => {
```

Comment:
A test that compares the return value of `words()` with a hard-coded array of the expected words would be useful in ensuring the method actually works. e.g `expect('two words'.words()).toEqual(['two', 'words'])`. You'd have several such tests for different edge cases.